crates/router/src/connector/adyen/transformers.rs

```diff
@@ -646,6 +654,11 @@ fn get_payment_method_data<'a>(
                 }
             }
         }
+        _ => Err(errors::ConnectorError::NotSupported {
```

Comment:
Avoid using underscore to handle other cases, using underscore we can find NotSupported only in runtime and not in compile time.